bin/run_terminology.sh

```diff
@@ -11,73 +11,34 @@ if [ -f /.dockerenv ]; then
   fi
 fi
 
-temp_folder="tmp/terminology"
-
-if [[ ! -f "$temp_folder/umls.zip" ]]; then
-  echo "The UMLS zip file must be manually downloaded and placed in the tmp/terminology folder."
-  echo "For more info, see the 'terminology support' section of the README."
-  exit 1
+./bin/prepare_terminology.sh 2019
```

Comment:
It would be awesome if we could have some sort of config file with the years we wanted so that we wouldn't have to make actual code changes to update these. I guess it would also have to manage AA/AB.